recipes/gdk-pixbuf/all/conanfile.py

```diff
@@ -130,7 +125,12 @@ def _configure_meson(self):
         defs["builtin_loaders"] = "all"
         defs["gio_sniffing"] = "false"
         defs["introspection"] = "enabled" if self.options.with_introspection else "disabled"
-        args=[]
+        args = []
+        # Workaround for https://bugs.llvm.org/show_bug.cgi?id=16404
+        # Ony really for the purporses of building on CCI - end users can
+        # workaround this by appropriately setting global linker flags in their profile
+        if self.settings.compiler == "clang":
+            args.append('-Dc_link_args="-rtlib=compiler-rt"')
```

Comment:
I believe that somehow you need to tell conan about the dependency on compiler-rt, so that the generated pkgconfig file will add compiler-rt as a dependency. Otherwise libraries that link against gdk-pixbuf might get unresolved symbols.